packages/react-core/src/components/FileUpload/FileUploadField.tsx

```diff
@@ -114,8 +114,8 @@ export const FileUploadField: React.FunctionComponent<FileUploadFieldProps> = ({
 
   ...props
 }: FileUploadFieldProps) => {
-  const onTextAreaChange = (newValue: string) => {
-    onTextChange?.(newValue);
+  const onTextAreaChange = (newValue: string, event: React.ChangeEvent<HTMLTextAreaElement>) => {
```

Comment:
Yeah I debated how I wanted to approach this. Since the `onChange` of TextArea that this gets passed into is still on the todo list of the param adding epic I went with just leaving this internal function in the order that it's in, but I could update the order here and just swap them in the `onChange`, either way works if you have a preference.

Reply:
Thanks for clarifying @wise-king-sullyman , in that case it makes sense to address it in the TextArea issue